src/util/meson.build

```diff
@@ -29,6 +29,7 @@ util = static_library(
   'ByteReverse.cxx',
   'format.c',
   'BitReverse.cxx',
+  'ExecOpen.cxx',
```

Comment:
This should not be in libutil, because libutil should be portable generic code without any dependencies. Better put it in libsystem.